src/backend/oracle-data-interface/src/main/java/ca/bc/gov/trafficcourtsonline/oracledatainterface/service/DisputeService.java

```diff
@@ -0,0 +1,44 @@
+package ca.bc.gov.trafficcourtsonline.oracledatainterface.service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import ca.bc.gov.trafficcourtsonline.oracledatainterface.model.Dispute;
+import ca.bc.gov.trafficcourtsonline.oracledatainterface.repository.DisputeRepository;
+
+@Service
+public class DisputeService {
+	@Autowired  
+	DisputeRepository disputeRepository;
+	
+	//getting all dispute records by using the method findAll() of CrudRepository  
```

Comment:
Ideally, **public** methods should have a javadoc comment rather than a **private** comment.
```suggestion
	/**
	 * Retrieves all {@link Dispute} records, delegating to CrudRepository  
	 * @return
	 */
```